src/views/commentView/index.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import Moment from "react-moment";
+import ReactTimeAgo from "react-time-ago";
+import { addComment } from "../../actions/commentActions/index";
+
+export class CommentView extends Component {
+    handleSubmit = (e) => {
+      e.preventDefault();
+      const comment = (e.target.elements.comment.value).trim();
+      const slugg = (e.target.elements.slugg.value);
+      if (comment) {
+        this.props.addComment(slugg, comment);
+      }
+    };
+    
+    isLoggedIn=() => localStorage.getItem("token") === null;
```

Comment:
We have a protected route method which ensures only logged in users can access a particular route please look into it

Reply:
Yes, I have looked into it. Thank you